src/utils/hints/oracles.js

```diff
@@ -0,0 +1,6 @@
+export const oraclesHints = {
+  oracleId: 'Identifier of Oracle that is similar to the address of the Account that registered the Oracle. The only difference is the prefix that will be ok_ instead of ak_. This means that each account can only host 1 oracle. It\'s not possible to manage multiple oracles using the same account.',
```

Comment:
I think most of the oracleId description could be simplified to something like:

_Unique oracle identifier inherited from the account address of the creator._

that implies that there can be only one oracle and that the mentioned hashes are related

Reply:
I like the proposal of @lukeromanowicz 👍

Reply:
👍

Reply:
Thanks, accepted